edit-post/store/selectors.js

```diff
@@ -16,8 +16,18 @@ export function getEditorMode( state ) {
  *
  * @return {string} Active sidebar panel.
  */
-export function getActivePanel( state ) {
-	return state.panel;
+export function getActiveEditorPanel( state ) {
+	return getPreference( state, 'activeSidebarPanel', {} ).editor;
+}
+
+/**
+ * Returns the current active plugin for the plugin sidebar.
+ *
+ * @param  {Object}  state Global application state
+ * @return {string}        Active plugin sidebar plugin
+ */
+export function getActivePlugin( state ) {
+	return getPreference( state, 'activeSidebarPanel', {} ).plugins;
```

Comment:
Minor: in the state tree we save as "plugins", but the selector uses "Plugin". Should we allow more than one plugin to be active? If yes the selector should be renamed to getActivePlugins if not the property should be renamed to plugin.

Reply:
@jorgefilipecosta That's a good question. I think because this function is sidebar specific (using 'activeSidebarPanel' in the getPreference function), there could only be one plugin active at any given time. I'll rename the property accordingly.